restricted_site_access.php

```diff
@@ -114,9 +116,25 @@ private static function set_option_defaults() {
 			return;
 		}
 
+		if ( defined( 'RSA_IS_NETWORK' ) && RSA_IS_NETWORK ){
+			self::$rsa_network_mode = get_site_option( 'rsa_mode' );
```

Comment:
You are checking this variable elsewhere which might not be defined. This will throw a notice.